### PR TITLE
styling added

### PR DESCRIPTION
**Pull request by Bearandbird**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,24 +3,23 @@
...
+    <link href="css/index.css" rel="stylesheet">
+</head>
 
- <style>
-    body {background-color: darkorange;}
-    h1 {color:honeydew}
-    p {color:darkslateblue}
+<style>
+   
  </style>
```
Comment (toma63) [inline index.html]: Just remove this.

File: index.html [hunk 2]
```diff
@@ -3,24 +3,23 @@
 
 <head> 
     <title> About Amber Hinojos </title>
+    <link href="css/index.css" rel="stylesheet">
+</head>
```
Comment (toma63) [inline index.html:7]: Consider adding google fonts as in the extra items for the assignment.

**Review by toma63 — CHANGES_REQUESTED** [review]
Nice design.  It looks like some of the lesson requirements were not met.

These apply to sections, not headings:
 Add spacing (padding/margin) between sections
 Change the alignment of the content of one of your sections
 
  Transform the "Experience" list items into styled blocks
  is calling for the display type to be block

**Comment by Bearandbird** [comment]
> Nice design. It looks like some of the lesson requirements were not met.
> 
> These apply to sections, not headings: Add spacing (padding/margin) between sections Change the alignment of the content of one of your sections
> 
> Transform the "Experience" list items into styled blocks is calling for the display type to be block

-Thank you- I think it's all fixed now.

**Pull request merged**